src/memote/experimental/essentiality.py

```diff
@@ -87,7 +87,6 @@ def evaluate(self, model):
             essen = single_gene_deletion(
                 model, gene_list=self.data["gene"], processes=1)
         essen["gene"] = [list(g)[0] for g in essen.index]
-        essen.index = essen["gene"]
```

Comment:
Is this change to satisfy the updated pandas?

Reply:
Yes, it now complains when the index and a column have the same name. Another solution would be to just give it a different name.